astropy/units/tests/test_utils.py

```diff
@@ -3,11 +3,15 @@
 Test utilities for `astropy.units`.
 """
 
+# ruff: noqa: FLY002
```

Comment:
This Ruff rule was suggesting that instead of building strings from hardcoded substrings using `str.join()` it would be better to hardcode the entire string. That is often good advice, but writing out multiline strings is always a bit awkward and here I do think using `"\n".join()` is the least bad option.